CreeDictionary/CreeDictionary/templates/CreeDictionary/query-help.html

```diff
@@ -1,49 +1,57 @@
 {% extends 'CreeDictionary/base.html' %}
-
+<!-- -->
 {% load creedictionary_extras %}
-
+<!-- -->
```

Comment:
Why the empty comments?

Reply:
I have been using [prettier](https://prettier.io) as an auto-formatter for my Django templates. It’s designed for regular HTML, and it mostly works, so long as I add empty comments between things that should be on their own lines.